test/libsolidity/syntaxTests/parsing/visibility_specifiers.sol

```diff
@@ -3,15 +3,12 @@ contract c {
     uint internal b;
     uint public c;
     uint d;
-    function f() {}
+    function f() public {}
```

Comment:
the function should be removed.

Reply:
Shouldn't we still test parsing the ``public`` visibility specifier for functions here?

Reply:
Sure, but it's done below.

Reply:
Ah sorry, didn't get that that line was removed.